flow-model-generator/modules/flow-model-generator-core/src/main/java/io/ballerina/flowmodelgenerator/core/Constants.java

```diff
@@ -71,4 +71,27 @@ private NaturalFunctions() {}
         public static final String ICON =
                 "https://gist.github.com/user-attachments/assets/903c5c16-7d67-4af8-8113-ce7c59ccdaab";
     }
+
+    // Constants used for AI
+    public static final class Ai {
```

Comment:
Shouldn't this be 
```suggestion
    public static final class AI {
```
?

Reply:
For the class name we normally use Pascal case even for abbreviated names. Hence following the same here.